dfusion_rust_core/src/util.rs

```diff
@@ -0,0 +1,42 @@
+use std::str::FromStr;
+use web3::types::U256;
+
+pub fn u128_to_u256(x: u128) -> U256 {
+    U256::from_big_endian(&x.to_be_bytes())
+}
+
+pub fn u256_to_u128(x: U256) -> u128 {
+    u128::from_str(&x.to_string()).unwrap()
+}
```

Comment:
Were these moved from somewhere else? I feel like I've seen them before.

Reply:
They were moved from `driver` crate to `dfusion_core` crate as it was needed in both now.